packages/a8c-mc-stats/composer.json

```diff
@@ -5,7 +5,7 @@
 	"license": "GPL-2.0-or-later",
 	"require": {},
 	"require-dev": {
-		"phpunit/phpunit": "^5.7 || ^6.5 || ^7.5"
+		"yoast/phpunit-polyfills": "^0.2"
```

Comment:
this yoast package is a nice find. Yay OSS.

Reply:
You're very welcome :smile: